CONTRIBUTING.md

```diff
@@ -24,34 +24,34 @@ provided for customers and users. So all code modifications SHOULD be submitted
 - Fork the repository on GitHub.
 - Read the README.md and INSTALL.md for project information and build instructions.
 
-For those who just get in touch with this project recently, here is a proposed contributing [tutorial](https://github.com/leonwanghui/installation-note/blob/master/opensds_fork_contribute_tutorial.md).
+For those who just get in touch with this project recently, here is a proposed contributing [tutorial](https://github.com/leonwanghui/installation-note/blob/master/soda_fork_contribute_tutorial.md).
 
 ## Contribution Workflow
 
 ### Code style
 
-The coding style suggested by the Golang community is used in opensds. See the [doc](https://github.com/golang/go/wiki/CodeReviewComments) for more details.
+The coding style suggested by the Golang community is used in soda. See the [doc](https://github.com/golang/go/wiki/CodeReviewComments) for more details.
 
-Please follow this style to make opensds easy to review, maintain and develop.
+Please follow this style to make soda easy to review, maintain and develop.
 
 ### Report issues
 
 A great way to contribute to the project is to send a detailed report when you encounter an issue. We always appreciate a well-written, thorough bug report, and will thank you for it!
 
 When reporting issues, refer to this format:
 
-- What version of env (opensds, os, golang etc) are you using?
+- What version of env (soda, os, golang etc) are you using?
 - Is this a BUG REPORT or FEATURE REQUEST?
 - What happened?
 - What you expected to happen?
 - How to reproduce it?(as minimally and precisely as possible)
 
 ### Propose PRs
 
-- Raise your idea as an [issue](https://github.com/opensds/opensds/issues)
-- If it is a new feature that needs lots of design details, a design proposal should also be submitted [here](https://github.com/opensds/design-specs/pulls).
+- Raise your idea as an [issue](https://github.com/soda/soda/issues)
```

Comment:
The GitHub repository soda/soda will not work. Please check and replace with appropriate GitHub repository. I think it should be [https://github.com/sodafoundation/dock/issues](https://github.com/sodafoundation/dock/issues)